addon/ng2/utilities/ast-utils.ts

```diff
@@ -52,3 +83,40 @@ export function insertAfterLastOccurrence(nodes: ts.Node[], toInsert: string,
   let lastItemPosition: number = lastItem ? lastItem.end : fallbackPos;
   return new InsertChange(file, lastItemPosition, toInsert);
 }
+
+/**
+* Custom function to insert component (component, pipe, directive)
+* into NgModule declarations. It also imports the component. 
+* @param modulePath
+* @param classifiedName 
+* @param importPath
+* @return Promise
+*/
+export function importComponent(modulePath: string, classifiedName: string, 
+    importPath: string): Promise<void> {
+  let source: ts.SourceFile = this.getSource(modulePath);
+
+  let importNode: ts.Node = 
+    this.findNodesByText(source, 'import').pop();
+  let iPos: ts.LineAndCharacter =
+    source.getLineAndCharacterOfPosition(importNode.getEnd());
+  let iLine: number = iPos.line + 1;
+  let iStart: number = source.getPositionOfLineAndCharacter(iLine, 0);
+  let iStr: string = `import { ${classifiedName} } from ${importPath}\n`;
+  let changeImport: InsertChange = new InsertChange(modulePath, iStart, iStr);
+
+  return changeImport.apply().then(() => {
+    source = this.getSource(modulePath);
+    let declarationsNode: ts.Node = 
+      this.findNodesByText(source, 'declarations').shift();
```

Comment:
Unfortunately, this will fail on a number of things, including an `import {declarations} from 'x'` and `var declarations = foo;`. I would suggest looking for the `@NgModule` call, take the first argument, then look for the declarations key. Also, please make sure the declarations is actually there (maybe it's not) and an array declaration.